tests/conftest.py

```diff
@@ -421,3 +431,45 @@ def archive():
 
     except Exception as e:
         warnings.warn(str(e))
+
+
+# Get the flat, fringe, and linearity for
+# a couple of DECam chip sand filters
+# Need session scope; otherwise, things
+# get mixed up when _get_default_calibrator
+# is called from within another function.
+@pytest.fixture( scope='session' )
```

Comment:
Generally I think when you are pulling external resources, session scope is the way to go...